backend/src/main/java/com/votogether/domain/member/dto/MemberDetailRequest.java

```diff
@@ -0,0 +1,20 @@
+package com.votogether.domain.member.dto;
+
+import com.votogether.domain.member.entity.Gender;
+import io.swagger.v3.oas.annotations.media.Schema;
+import jakarta.validation.constraints.Max;
+import jakarta.validation.constraints.Min;
+import jakarta.validation.constraints.NotNull;
+
+@Schema(description = "회원 상세 정보 수정 요청")
+
+public record MemberDetailRequest(
+        @Schema(description = "성별", example = "MALE")
+        Gender gender,
+        @Schema(description = "출생년도", example = "2000")
+        @NotNull(message = "출생년도는 빈 값일 수 없습니다.")
+        @Min(value = 4, message = "출생년도는 4자리만 가능합니다.")
+        @Max(value = 4, message = "출생년도는 4자리만 가능합니다.")
+        Integer birthYear
```

Comment:
`P3`
```suggestion
        @Schema(description = "성별", example = "MALE")
        Gender gender,
        
        @Schema(description = "출생년도", example = "2000")
        @NotNull(message = "출생년도는 빈 값일 수 없습니다.")
        @Min(value = 4, message = "출생년도는 4자리만 가능합니다.")
        @Max(value = 4, message = "출생년도는 4자리만 가능합니다.")
        Integer birthYear
```
이 부분도 컨벤션으로 얘기하면 좋을 것 같은데 각 필드마다 공백을 주는 것이 어떨까요?

자바 `validation` 어노테이션까지 포함되니 가독성이 떨어지는 부분이 있는 것 같아요!

Reply:
동의합니다 수정할게요~